contracts/DataFeed.sol

```diff
@@ -77,27 +82,42 @@ contract DataFeed is usingOraclize, DestructibleModified {
     }
   }
 
+  // Assumes that the result is a raw JSON object with exactly 2 fields: 
```

Comment:
NAV service outputs the following:

GDAX
`{"totalPortfolioValue":VALUE,"USDETH":VAL,"USDBTC":VAL,"USDLTC":VAL}`

GEMINI
`{"totalPortfolioValue":VALUE,"USDETH":VAL,"USDBTC":VAL}`

Values may also be more than 2 decimal places